src/v2/Apps/Conversation/Components/OrderUpdate.tsx

```diff
@@ -55,16 +55,20 @@ export const OrderUpdate: React.FC<OrderUpdateProps> = ({
       return null
     }
   } else if (event.__typename === "CommerceOrderStateChangedEvent") {
-    const { state, stateReason } = event
-    if (state === "APPROVED") {
+    const { orderUpdateState } = event
+    if (orderUpdateState === "offer_approved") {
       color = "green100"
       message = `Offer Accepted`
-    } else if (state === "CANCELED" && stateReason?.includes("_rejected")) {
+    } else if (orderUpdateState === "offer_rejected") {
       color = "red100"
       message = `Offer Declined`
-    } else if (state === "CANCELED" && stateReason?.includes("_lapsed")) {
+    } else if (orderUpdateState === "offer_lapsed") {
       color = "black60"
       message = `Offer Expired`
+    } else if (orderUpdateState === "buy_submitted") {
```

Comment:
maybe worth to check what happens if an order is canceled, we still show the `You purchased this artwork` for this case but in reality, the order was canceled or refunded.

Reply:
We will check this with QA and create a new ticket for all the states.